# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: ab23423c7353
# Full Hash: ab23423c7353fb44765f39233fd660fa9cb75fec
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-24 18:03 +0000
# Regressor Bug: 1968890
# File Overlap Count: 2
# Overlapping Files: js/loader/ModuleLoadRequest.cpp, js/loader/ModuleLoaderBase.cpp
# Description:
#   Bug 1968890 - Part 3: Call ModuleErrored in OnFetchFailed. r=jonco
# ==============================================================================

diff -r 06ec60266948 -r ab23423c7353 js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Wed Sep 24 18:03:29 2025 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Wed Sep 24 18:03:29 2025 +0000
@@ -107,10 +107,6 @@
   MOZ_ASSERT(IsFetching() || IsPendingFetchingError());
 
   mModuleScript = mLoader->GetFetchedModule(ModuleMapKey(mURI, mModuleType));
-  if (IsErrored()) {
-    ModuleErrored();
-    return;
-  }
 }
 
 void ModuleLoadRequest::LoadFailed() {